lib/game.rb

```diff
@@ -26,18 +26,9 @@ def computer_placement
     @computer.theorize_and_place_ships(ships)
   end
 
-  def convert_input_to_coords()
-    valid_coord = false
-
-    until valid_coord
-      coords = gets.chomp.split
-
-      if (coords.all? { |coord| coord.split("").length == 2 })
-        return coords.map { |coord| coord.upcase}
-      else
-        print "Please enter valid coordinates! \n"
-      end
-    end
+  def convert_input_to_coords
```

Comment:
### What does this do?
I believe this was leftover code that no longer was needed here.